manifest.json

```diff
@@ -7,6 +7,32 @@
               "48": "icons/icon48.png",
              "128": "icons/icon128.png" },
   "minimum_chrome_version": "69.0",
+  "commands": {
+    "jump-back-tab": {
+      "description": "Jump back in the tab stack."
+    },
+    "jump-forward-tab": {
+      "description": "Jump forward in the tab stack."
+    },
+    "open-vomnibox": {
+      "description": "Open the vomnibox for the same tab."
+    },
+    "open-vomnibox-new-tab": {
+      "description": "Open the vomnibox for a new tab."
+    },
+    "open-vomnibox-tab": {
+      "description": "Open the vomnibox with tabs."
+    },
+    "open-vomnibox-bookmark": {
+      "description": "Open the vomnibox with history."
+    },
+    "open-vomnibox-bookmark-new-tab": {
+      "description": "Open the vomnibox with history."
+    },
+    "switch-to-previous-tab": {
+      "description": "Switch to the previously used tab."
+    }
+  },
```

Comment:
Just a personal opinion:

Maybe we can use command names of Vimium directly, like what I've done in https://github.com/gdh1995/vimium-c/blob/dc5e026083db67fc6f471c907e5ca37b7b5a5316/manifest.json#L38-L63 .

Then we may add new syntaxes to key mappings to assign options to such global "shortcuts" (like https://github.com/gdh1995/vimium-c/wiki/Trigger-commands-in-an-input-box#shortcuts)